pkg/proxy/backup.go

```diff
@@ -150,7 +151,34 @@ func (p *Proxy) BackupRestore(ctx context.Context, req *rpc.EngineBackupRestoreR
 
 	resp = &rpc.EngineBackupRestoreProxyResponse{}
```

Comment:
NIT: same here. don't need to init the response at the beginning.

Reply:
Updated.